go/builder.go

```diff
@@ -117,15 +119,23 @@ func (w *InternalBuilder) WriteBytes20(buf []byte, v [20]byte) {
 	if buf != nil {
 		WriteBytes20(buf[w.size:], v)
 	}
-	w.size += Offset(len(v))
+	w.size += FieldSizes[TypeBytes20]
```

Comment:
why?

Reply:
because this field is exactly the value of the calculation.